.github/workflows/deploy-reearth-flow-prod.yml

```
@@ -83,109 +69,34 @@
-    steps:
-      - uses: google-github-actions/auth@v2
-        with:
-          service_account: ${{ secrets.GCP_SERVICE_ACCOUNT }}
-          workload_identity_provider: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
-      - name: Configure docker
-        run: gcloud auth configure-docker asia-northeast1-docker.pkg.dev --quiet
-      - name: Log in to GitHub Container Registry
+      - name: Pull image
+        run: docker pull $WEB_IMAGE_NAME_GHCR
+      - name: Log in to DockerHub
         uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)